src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/register/AttachedRegisterPage.java

```diff
@@ -48,6 +53,14 @@ public AttachedRegisterPage() {
     return !wait.forElementVisible(submitButton).isEnabled();
   }
 
+  @Override public boolean isPasswordMasked() {
+    return passwordField.getAttribute("type").equals("password");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the "password" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1132)